_books/ion-1-1/src/text/encoding.md

```diff
@@ -0,0 +1,10 @@
+# Ion 1.1 Text Encoding
+
+The Ion text encoding is a stream of UTF-8 encoded text.
+It is intended to be easy to read and write by humans in any text-based application.
+
+Whitespace is insignificant and is only required where necessary to separate tokens, and C-style comments 
+(either block or in-line) are treated as whitespace (and are not part of the binary encoding).
+
+A text Ion 1.1 stream begins with the Ion 1.1 [version marker](../todo.md) (`$ion_1_1`) followed by a series of
```

Comment:
In the binary section I think I linked to the glossary for "Ion version marker". Do you think it needs its own page or section somewhere?

Reply:
I think we need a section that defines the semantics of an Ion stream. By that I mean a chapter that covers things like IVMS, system values vs user values, and `$ion_literal`.